service/src/main/java/uk/gov/hmcts/reform/fpl/service/document/ManageDocumentService.java

```diff
@@ -278,6 +281,58 @@
         return setDateTimeUploadedOnSupportingEvidence(currentSupportingDocuments, previousSupportingDocuments);
     }
 
+    public List<Element<SupportingEvidenceBundle>> getRespondentStatementFurtherEvidenceCollection(CaseData caseData,
```

Comment:
should it not go to ManageDocumentServiceLA if functionality exists only for LA

Reply:
Next story is to implement the same feature only for the admin. Hence why I placed this functionality here. Let's keep here.